StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/SavedPaymentMethods/SavedPaymentMethodsViewController.swift

```diff
@@ -120,14 +120,14 @@ class SavedPaymentMethodsViewController: UIViewController {
         self.isApplePayEnabled = isApplePayEnabled
         self.spmsCompletion = spmsCompletion
         self.delegate = delegate
-        if savedPaymentMethods.isEmpty {
+        if !savedPaymentMethods.isEmpty || isApplePayEnabled {
```

Comment:
We should probably use `shouldShowPaymentMethodCarousel` here?

Reply:
Good point, refactored a bit. We can't call it on `self` before `super.init`, but we can at least share the logic.